framework/src/controller/channels/event_emitter.js

```diff
@@ -92,7 +92,7 @@ class EventEmitterChannel extends BaseChannel {
 	 * @param {array} params - Params associated with the action
 	 * @return {Promise<string>} Data returned by bus.
 	 */
-	async invoke(actionName, params) {
+	invoke(actionName, params) {
```

Comment:
`invoke` has to be async?

Reply:
Ideally it has, but we cannot force to use it that way with the current status of our code. This is a key point to go forward with this issue. I had a talk with @lsilvs about it. Please, raise any concern in any case where removing the mandatory async use of this function creates a problem.

Reply:
Reverted.